src/v0/destinations/am/transform.js

```diff
@@ -499,6 +499,23 @@ function responseBuilderSimple(
           delete payload.user_properties.address.country;
         }
       }
+
+      if (!payload.user_id && !payload.device_id) {
+        logger.debug("Both userId and deviceId cannot be undefined");
+        throw new ErrorBuilder()
```

Comment:
Can we change to new error class convention, confirm what error class does this kind of error belong to & make relevant changes ?